tests/framework/web/UrlRuleTest.php

```diff
@@ -51,8 +51,8 @@ public function testParseRequest()
                 $request->pathInfo = $test[0];
                 $expected = $test[1];
                 $result = $rule->parseRequest($manager, $request);
-                if ($expected === false) {
-                    $this->assertFalse($result, "Test#$i-$j: $name");
+                if (empty($expected)) {
```

Comment:
why are these checks done with empty? I dont see any occurrence in the test execution in which an empty array is checked here? Only `false` or a filled `array`